content/_code-samples/freeze/py/set_global_freeze.py

```diff
@@ -0,0 +1,25 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountSet, AccountSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # Connect to testnet
+
+# Sender wallet object
+sender_wallet = generate_faucet_wallet(client)
+
+# Build accountset transaction to enable global freeze
+accountset = AccountSet(account=sender_wallet.classic_address,
+    set_flag=AccountSetFlag.ASF_GLOBAL_FREEZE)
+
+# Sign and submit transaction
+stxn = safe_sign_and_autofill_transaction(accountset, sender_wallet, client)
+stxn_response = send_reliable_submission(stxn, client)
+
+# Parse response for result
+stxn_result = stxn_response.result
+
+# Print result and transaction hash
+print(stxn_result["meta"]["TransactionResult"])
+print(stxn_result["hash"])
```

Comment:
Similar to above, I'd recommend adding a print statement upon successful completion of the script that summarizes what just happened into the commandline.